subcommands/keys/ca_create.go

```diff
@@ -78,7 +83,24 @@ func run(name string, arg ...string) string {
 	return string(out)
 }
 
+func getDeviceCaCommonName() string {
+	f, err := os.Open("create_device_ca")
+	subcommands.DieNotNil(err)
+	defer f.Close()
+
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		if strings.HasPrefix(scanner.Text(), "CN = ") {
+			return strings.TrimSpace(strings.Split(scanner.Text(), "=")[1])
+		}
+	}
+	err = scanner.Err()
+	subcommands.DieNotNil(err)
+	return ""
+}
```

Comment:
Just decide on your own... I think below should help.

IIUC the CN is always in a form `fio-<current-user-id>`.
We can use it to simplify this code like below by calling the server:
```
user, err := api.UserAccessDetails(factory, "self")
return "fio-" + user.PolisId
```